code/css/style.css

```diff
@@ -0,0 +1,198 @@
+/* style.css */
+
+/* setup */
+* {
+  box-sizing: border-box;
+}
+
+/* clear fix hack */
+.clearfix:after {
+  content: ".";
+  display: block;
+  clear: both;
+  line-height: 0;
+  height: 0;
+}
+
+.clear {
+  clear:both;
+}
+
+.alignleft {
+  float: left;
+}
+
+.alightright {
+  float: right;
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+/* lyout */
+.container {
+  width: 960px;
+  margin: 0 auto;
+}
+
+body {
+  color: ;
+  background: #A5B8A7;
+  font-family: Amatic SC;
+  font-size:;
+  font-weight:;
+  line-height:;
+}
+
+a {
+  text-decoration: none;
+}
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+header {
+  padding: 25px 0;
+  overflow:hidden;
+  background:#ED5A36;
+}
+
+header nav {
+  float: right;
+  margin-top: 15px;
+}
+
+header nav ul li {
+  display: inline-block;
+  margin: 0 0 0 40px;
+  font-weight: 900;
+  font-size:30px;
+}
+
+header nav ul li a {
+  color:#994A00;
+}
+
+header .logo p {
+  font-size: 30px;
+  color: #2E2B28;
+}
+
+#main {
+
+}
+
+#landing {
+  display: inline-block;
+  vertical-align: top;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+  background: url("images/whaleSample.png") no-repeat scroll center center;
+height: 100%;
+position: relative;
+width: 100%;
+border-bottom: 2px black solid;
+}
+
+#story {
+  display: inline-block;
+  vertical-align: top;
+  wisth: 100%;
+  border-bottom: 2px black solid;
+}
+
+#products {
+  display: inline-block;
+  vertical-align: top;
+  width:100%;
+  border-bottom: 2px black solid;
+}
+
+.eight_colums{
+
+}
+
+.one-forth{
+  float:left;
+  display:inline-block;
+  padding-left: 10px;
+  margin-bottom: 10px;
+  width:25%;
+
+
+}
+
+
+#follow_us {
+  display: inline-block;
+  vertical-align: top;
+  width:100%;
+  border-bottom: 2px black solid;
+}
+
+.six_columns{
+
+}
+
+.one-third {
+  float:left;
+  display: inline-block;
+  margin-left: 10px;
+  width:30%;
+}
+
+.alpha {
+  clear:left;
+}
+
+.omega {
```

Comment:
Any reason for these names? Would be a lot clearer to other developers (and probably yourself in the future) to name them something like `clear-right` and `clear-left`